package.json

```diff
@@ -58,6 +58,7 @@
     "gulp-util": "^3.0.7",
     "gzip-js": "~0.3.2",
     "jest": "^15.1.1",
+    "jest-cli": "^15.1.1",
```

Comment:
I think that just adding `jest-cli` is not enough.

The problem is we haven't synced `package.json` for a while, and we need to update it to match what we have on `master`. For example [we already have jest-cli@18 there](https://github.com/facebook/react/blob/1c43f9471773a5206279236c5e8b5b37d3e985cb/package.json#L64).

In general, in this PR the goal is to bring it as close as possible to master. We probably missed some commits at some point. There shouldn't be much intentional divergence between `15-stable` and `master`. So now that you found a set of changes that work, could you also reduce it so that it matches `master` whenever possible?